kibble/api/languages.go

```diff
@@ -64,6 +69,46 @@ func loadAllLanguagesFromApi(cfg *models.Config, site *models.Site) error {
 	return nil
 }
 
+// Returns an error if:
+// - default language is not set.
+// - default language is missing display name.
+// - no site languages are set.
+// - any site language is missing a display name.
+// - default language is not included in supported languages.
+func (l languagesV1) validate() error {
+	isZero := func(val string) bool {
+		return val == ""
+	}
+
+	if isZero(l.DefaultLanguage.Code) && isZero(l.DefaultLanguage.Label) && isZero(l.DefaultLanguage.Name) {
+		return fmt.Errorf("DefaultLanguage not set")
```

Comment:
Sorry. I forgot to mention this.
Go recommends using errors.New() over fmt.Errorf(), if there is no formatting.

Reply:
![Argh!](https://c.tenor.com/No8u0Yip0lwAAAAC/so-pissed.gif)

Reply:
:grimacing: